src/Build/Logging/SerialConsoleLogger.cs

```diff
@@ -462,6 +512,22 @@ public override void TaskFinishedHandler(object sender, TaskFinishedEventArgs e)
             }
         }
 
+        /// <summary>
+        /// Finds the LogOutPropterty string to be printed in messages
+        /// </summary>
+        /// <param name="e"> the build event where the LogOutPutProperty string will be added</param>
+        internal void PropertyMapping(LazyFormattedBuildEventArgs e)
+        {
+            string logOutputProperties = "";
```

Comment:
tiny nit:
```suggestion
            string logOutputProperties = string.Empty;
```

Reply:
Just a style thing? As these generate the same code.

Reply:
I believe "" allocates a new empty string, whereas all string.Empty calls use the same empty string, so it also saves (a tiny amount of) space.

Reply:
I [don't think that's the case](https://sharplab.io/#v2:EYLgtghgzgLgpgJwDQxNMAfAAgJgIwCwAUFgMwAEu5AwuQN7HlOUVYAs5AsgBQCU9jZkKx4ADOSh5yAXnIBlGAgCWAOwDmAOgCiYAA4wAngG5BQpiICc3SbxNEhAX1PlnZSh044+A+2fNiJKVkAImC7P38rG3DmJyIHIA===)

At the time I worked in this code, we did believe it was the case, and used String.Empty consistently. It may or may not have been true then (~2005). I think it is now a matter of style only.

Reply:
That seems convincing to me. Looking around online, someone suggested that "" could hide a zero-length space, but I don't think we have to worry about that.😄 

It would still be good to match original style, though.

Reply:
Yes, for sure. I was used to String.Empty, but in my current project we use "" (although I see it's not in the [style guide](https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/coding-style.md))